### PR TITLE
Update MODULEPATH for hera following Rocky8 upgrade

### LINKED ISSUE #24: Hera Rocky 8 migration
Hera is undergoing migration to Rocky 8.
The utilities under this project will need to be built and tested on Hera with Rocky 8

### PR DESCRIPTION
**Pull request by aerorahul**
Closes #24

### CODE REVIEW

**Comment by HenryRWinterbottom** [comment]
LGTM.

**Review by HenryRWinterbottom — APPROVED** [review]

**Review by DavidHuber-NOAA — APPROVED** [review]

**Comment by aerorahul** [comment]
@HenryWinterbottom-NOAA 
This branch was installed under `/scratch1/NCEPDEV/global/glopara/git/prepobs/feature-hera-rocky8`.
Was that installation tested with cycling in global-workflow?

File: modulefiles/prepobs_hera.lua [hunk 1]
```diff
@@ -2,7 +2,9 @@ help([[
 Load environment to build prepobs on Hera
 ]])
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev/install/modulefiles/Core")
+--prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev/install/modulefiles/Core")
```
Comment (KateFriedman-NOAA) [inline modulefiles/prepobs_hera.lua]: Why leave the old `MODULEPATH` here?

**Review by KateFriedman-NOAA — CHANGES_REQUESTED** [review]

**Comment by HenryRWinterbottom** [comment]
> @HenryWinterbottom-NOAA Was this branch installed under `/scratch1/NCEPDEV/global/glopara/git/prepobs/feature-hera-rocky8` tested with cycling in global-workflow?

No. At least not yet.

**Comment by aerorahul** [comment]
> > @HenryWinterbottom-NOAA Was this branch installed under `/scratch1/NCEPDEV/global/glopara/git/prepobs/feature-hera-rocky8` tested with cycling in global-workflow?
> 
> No. At least not yet.

Apologies.  I did not frame the q. correctly.  I have corrected the comment.
The branch was installed under the location `/scratch1/NCEPDEV/global/glopara/git/prepobs/feature-hera-rocky8`
The q. is, was this installation tested with cycling in the global-workflow?

**Comment by HenryRWinterbottom** [comment]
@aerorahul What specific CI test includes this? I can test right now if I haven't already.

**Comment by aerorahul** [comment]
> @aerorahul What specific CI test includes this? I can test right now if I haven't already.

any cycled test with GSI will test this.
I suggest [C96_atm3DVar.yaml](https://github.com/NOAA-EMC/global-workflow/blob/develop/ci/cases/pr/C96_atm3DVar.yaml)

**Comment by HenryRWinterbottom** [comment]
@aerorahul Which global-workflow submodule is prepobs a part of? I can build and run the 3DVAR test again.

**Comment by aerorahul** [comment]
> @aerorahul Which global-workflow submodule is prepobs a part of? I can build and run the 3DVAR test again.

See https://github.com/NOAA-EMC/global-workflow/issues/2329#issuecomment-1977306052

**Comment by HenryRWinterbottom** [comment]
Thanks.

File: modulefiles/prepobs_hera.lua (hunk 1 above)
Comment (aerorahul) [inline modulefiles/prepobs_hera.lua]:
```suggestion
```
Remove commented out lines.

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good, thanks @aerorahul !

**Pull request merged**